### ISSUE TITLE
Install problem LightZone 5.0.0 beta1- Windows 10 x64 22H2

### ISSUE THREAD

**Issue by sattvakarma** [ktgw0316/LightZone#299]
Hello,

I am unable to install LightZone 5.0.0 beta1 on my Windows 10 x64 22H2.

Machine is a Dell Optiplex 3020 with 16 Gb RAM and a 256 MB SSD.

Attaching error message pic with this post.

Kindly advise on suitable course of action.

Regards.
Sattva

![photo_2023-07-24_13-59-20](https://github.com/ktgw0316/LightZone/assets/89531217/eca2de81-3514-4c5a-b7ad-8a0b3992fcce)

**Comment by jedd** [comment]
Hey @sattvakarma  -- is this your machine, or a work computer?  Or more specifically, do you have administrator access to this computer?

On first run of the installer the UAC pop-up should ask if you're happy to run the installer.  If you're not getting that, and/or your machine is administered by your work, this may not be something you can install without talking to your network / system administrator.

**Comment by sattvakarma** [comment]
@jedd 

Personal home PC. And i tried running the installer as administrator also using the right click option on the installer file. Same results. As mentioned by you, the first warning pop up, i choose the option to run anyway.

What more can i try to get LZ running?

For your information, i just installed the new version of Inkscape without any problems whatsoever.

**Comment by jedd** [comment]
Hi @sattvakarma  - oh, okay, thank you for confirming. I'm primarily a Linux user, so only bump into these kinds of issues infrequently.  The notification there - 'an administrator has blocked you from running this app' - makes it sound like something else is being enforced - an anti-virus package or similar? - even though the title on that dialog is UAC.

I don't expect there's been any regression on the latest release, so I'll defer to someone with more Microsoft Windows experience.

I note the documentation at https://learn.microsoft.com/en-us/windows/security/application-security/application-control/user-account-control/settings-and-configuration?tabs=intune suggests there's a whole lotta knobs you can play with (perhaps under Control Panel / UAC Settings ?)

**Comment by sattvakarma** [comment]
Hello @jedd 

Thank you for your assistance. Appreciate it.

I have tried most of the 'knobs' and 'sliders' :)) and also searched on Google and Windows documentation online. And nothing has changed. 

I am still not able to install LightZone 5.0.0 beta1. 

I am hopeful that the lead developer @ktgw0316 will get to read this post and see if there is anything that he can do to replicate my problem and solve this issue.

Regards,
Sattva

**Comment by ktgw0316** [comment]
Hello @sattvakarma,

I _guess_ that this issue is caused by an expired code-signing certificate (LightZone is still using an old certificate issued for lightcrafts inc.).  To be honest, I don't feel like to pay several hundred dollars for certificate authority to update it and check my guess is correct.  Not only Windows, but also macOS requires another paid certificate to distribute packages.  The world is becoming a hard place to live in for non-profit OSS developers. :(

Anyway, I'll find a workaround by next beta release.  My idea for now is to provide a batch file to execute jar file with Java runtime already included in LightZone folder.

**Comment by MJacal** [comment]
How I did it on Win10:

"Unblocking" the installer and running as admin didn't work, but after I have restarted my file manager as admin, the problem was gone.

(If I remember correctly, on a Win11 laptop I had to additionally change/disable some security settings.)

Be well!

**Comment by sattvakarma** [comment]
Hello @ktgw0316 ,

Thank you for your insights. Appreciate it!

Paid certificates can be deferred. And i agree with you, big software corporations are used to living of the fat of the land...so that habit is not going to change soon. Or change for that matter.

Please guide me when you have a suitable workaround to solve this issue. 

Thank you for your invaluable contributions! 

Regards and best wishes,
Sattva

**Comment by sattvakarma** [comment]
Hello @MJacal ,

Thank you for your tip.

But for me it does not work. Same error as before. No change.

Regards,
Sattva

**Comment by MJacal** [comment]
Windows security -> App & and browser control -> Reputation-based protection -> set this to Off, at least temporarily.

It tends to block less-known (and sometimes major) foss programs, even more development versions.

**Comment by sattvakarma** [comment]
Hello @MJacal ,

Tried your suggestion. Still no change in problem. Same error message as before.

Regards,
Sattva

**Comment by sfink16** [comment]
> Hello @sattvakarma,
> 
> I _guess_ that this issue is caused by an expired code-signing certificate (LightZone is still using an old certificate issued for lightcrafts inc.). To be honest, I don't feel like to pay several hundred dollars for certificate authority to update it and check my guess is correct. Not only Windows, but also macOS requires another paid certificate to distribute packages. The world is becoming a hard place to live in for non-profit OSS developers. :(
> 
> Anyway, I'll find a workaround by next beta release. My idea for now is to provide a batch file to execute jar file with Java runtime already included in LightZone folder.

I can confirm that the old Lightscraft certificate expired on 6/30/23.  It was supposed to expire in 2033 but it was manually revoked by the owner on the date I just listed.  This is why LZ beta can't be installed on windows at this time.  I can also confirm that I could successfully install 4.2.4 on my windows 10 device.  I thought that that is interesting.

Steve

**Comment by sattvakarma** [comment]
Hello @sfink16 ,

Thank you for the confirmation. Appreciate it.

Yes, once i was unable to install LZ 5.0.0 beta1, i re-installed LZ 4.2.4 with no issues.

Will wait for @ktgw0316 to come up with a solution. Even if it is a temporary install hack.

Really look forward to trying out LZ 5.0.0 beta1 :))

Thank you once again and best regards,
Sattva

**Comment by sfink16** [comment]
> Hello @sfink16 ,
> 
> Thank you for the confirmation. Appreciate it.
> 
> Yes, once i was unable to install LZ 5.0.0 beta1, i re-installed LZ 4.2.4 with no issues.
> 
> Will wait for @ktgw0316 to come up with a solution. Even if it is a temporary install hack.
> 
> Really look forward to trying out LZ 5.0.0 beta1 :))
> 
> Thank you once again and best regards, Sattva

I'm glad I helped, even for a little bit @sattvakarma .  You might want to try the WSL ([Windows For Linux](https://www.freecodecamp.org/news/how-to-run-linux-apps-on-windows-10-11-using-wsl/)), which runs Ubuntu by default.  LZ 5.0 Beta does work on my Ubuntu machine.  You simply use the command line wsl --install.  Unfortunately I got stuck with a BIOS issue after I tried to run the newly created Ubuntu app.  Perhaps you'll have more success than I did with the higher specs device you're using.
Steve

**Comment by user-by** [comment]
Need LightZone.exe file for to use with GIMP LightZone Plugin.

**Comment by sfink16** [comment]
Is there a new comment that I'm not seeing?  I can say that the version of Lightzone is now 5.0.0~beta2-0, rather than this older version, if that helps.
